### PR TITLE
Added basic views, and fixed some bugs.

### PR DESCRIPTION
**Pull request by Zawgor24**
Added basic views, and some bootstrap on User_show(profile) and Post_index.
 The project has been tested, and corrected some mistakes at controller actions.

### CODE REVIEW

File: app/assets/stylesheets/active_admin.scss [hunk 1]
```diff
@@ -1,2 +1,17 @@
+// SASS variable overrides must be declared before loading up Active Admin's styles.
```
Comment (IlyaUmanets) [inline app/assets/stylesheets/active_admin.scss]: Remove comment please

File: app/models/article.rb [hunk 2]
```diff
@@ -7,4 +7,6 @@ class Article < ApplicationRecord
 
   validates :title, :body, presence: true
   validates :title, length: { maximum: 20 }
+
+  mount_uploader :avatar, AvatarUploader
```
Comment (IlyaUmanets) [inline app/models/article.rb]: AvatarUploader is not the best name. I propose you to name it like `ImageUploader`

File: app/views/articles/edit.html.slim [hunk 3]
```diff
@@ -0,0 +1,6 @@
+h1 
+  | Edit `#{@article.title}` news
+
+= render 'form'
```
Comment (IlyaUmanets) [inline app/views/articles/edit.html.slim:4]: `render :form`

File: app/views/articles/show.html.slim [hunk 4]
```diff
@@ -0,0 +1,17 @@
+h1
+  | News №#{@article.id}
+
+= image_tag(@article.avatar_url(:thumb)) if @article.avatar.present?
```
Comment (IlyaUmanets) [inline app/views/articles/show.html.slim]: I propose you to move this line to helper. Helper is a method you can use in a view.

Let me know if you need any help to create a helper method

**Review by IlyaUmanets — CHANGES_REQUESTED** [review]

**Review by IlyaUmanets — CHANGES_REQUESTED** [review]
You don't need to push uploaded images. Ignore them please

File: app/assets/stylesheets/active_admin.scss (hunk 1 above)
Comment (Zawgor24) [reply app/assets/stylesheets/active_admin.scss]: Removed all comments.

File: app/models/article.rb (hunk 2 above)
Comment (Zawgor24) [reply app/models/article.rb]: Changed Avatar uploader to Image.

File: app/views/articles/show.html.slim (hunk 4 above)
Comment (Zawgor24) [reply app/views/articles/show.html.slim]: Added show_image method to application_helper.rb

**Review by IlyaUmanets — APPROVED** [review]

**Pull request merged**